core/trino-main/src/main/java/io/trino/connector/system/MaterializedViewSystemTable.java

```diff
@@ -57,7 +56,7 @@
             .column("storage_catalog", createUnboundedVarcharType())
             .column("storage_schema", createUnboundedVarcharType())
             .column("storage_table", createUnboundedVarcharType())
-            .column("is_fresh", BOOLEAN)
+            .column("freshness", createUnboundedVarcharType())
```

Comment:
Naive question: Can we change freely without worrying of breaking changes the system tables?

Reply:
Every change like this can break something. However, I don't see any better way to approach that.
Keeping `is_fresh` column as deprecated would force me to determine whether UNKNOWN freshness is "true" or "false", but it's neither. This could also break tools.
Explicit breakage is often better than a hidden one.